src/web/client/common/remoteSaveProvider.ts

```diff
@@ -7,25 +7,18 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import * as vscode from 'vscode';
 import { getHeader } from './authenticationProvider';
-import { columnExtension, FILE_EXTENSION_REGEX } from './constants';
 import { ERRORS, showErrorDialog } from './errorHandler';
+import { SaveEntityDetails } from './portalSchemaInterface';
 
-export async function saveData(accessToken: string, requestUrl: string, fileUri: vscode.Uri, entity: string, saveDataMap: Map<string, string>, value: string) {
+export async function saveData(accessToken: string, requestUrl: string, fileUri: vscode.Uri, entity: string, saveDataMap: Map<string, SaveEntityDetails>, value: string) {
```

Comment:
saveData function should not have entity as a parameter. It should fetch entity from the saveDataMap